### PR TITLE
[zh] translation for the run-app section

### LINKED ISSUE #29329: [zh] Tasks files to sync for 1.22
Numbers in the following list may be not up to date. Please run `./scripts/lsync.sh` again before working on each of them.

- [x] (1) run-app  @steven-my  #29451
```
5	6	content/zh/docs/tasks/debug-application-cluster/debug-running-pod.md
1	1	content/zh/docs/tasks/debug-application-cluster/debug-cluster.md
**removed**	content/zh/docs/tasks//debug-application-cluster/logging-stackdriver.md
1	1	content/zh/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md
3	3	content/zh/docs/tasks/run-application/delete-stateful-set.md
1	1	content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
11	8	content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md
```

- [x] (2) Kubectl install  @steven-my  #29509

```
45	7	content/zh/docs/tasks/tools/install-kubectl-windows.md
81	2	content/zh/docs/tasks/tools/install-kubectl-macos.md
57	6	content/zh/docs/tasks/tools/install-kubectl-linux.md
**removed**	content/zh/docs/tasks//tools/included/install-kubectl-gcloud.md
```

- [x] (3) objects & network  @steven-my    #29772

```
3	3	content/zh/docs/tasks/manage-kubernetes-objects/kustomization.md
4	4	content/zh/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md
1	1	content/zh/docs/tasks/network/validate-dual-stack.md
130	0	content/zh/docs/tasks/network/customize-hosts-file-for-pods.md
```

- [x] (4) pod container @superleo  #29821 

```
6	4	content/zh/docs/tasks/configure-pod-container/quality-service-pod.md
8	5	content/zh/docs/tasks/configure-pod-container/configure-service-account.md
1	1	content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md
21	6	content/zh/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md
20	1	content/zh/docs/tasks/configure-pod-container/security-context.md
2	2	content/zh/docs/tasks/configure-pod-container/configure-runasusername.md
0	6	content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md
```

- [x] (5) windows  @superleo  #29798

```
71	18	content/zh/docs/tasks/configure-pod-container/configure-gmsa.md
```

- [x] (6) admin 1   @steven-my   #29797

```
22	23	content/zh/docs/tasks/administer-cluster/enabling-service-topology.md
1	0	content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
2	2	content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md
26	2	content/zh/docs/tasks/administer-cluster/safely-drain-node.md
17	26	content/zh/docs/tasks/administer-cluster/kubelet-config-file.md
```

- [x] (7) admin 2   @steven-my  #29814

```
1	1	content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md
1	0	content/zh/docs/tasks/administer-cluster/declare-network-policy.md
1	1	content/zh/docs/tasks/administer-cluster/extended-resource-node.md
29	37	content/zh/docs/tasks/administer-cluster/reserve-compute-resources.md
1	1	content/zh/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy.md
39	25	content/zh/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md
```

- [x] (8) admin 3  @steven-my #29815

```
3	23	content/zh/docs/tasks/administer-cluster/coredns.md
1	1	content/zh/docs/tasks/administer-cluster/encrypt-data.md
32	23	content/zh/docs/tasks/administer-cluster/reconfigure-kubelet.md
4	3	content/zh/docs/tasks/administer-cluster/namespaces.md
1	1	content/zh/docs/tasks/administer-cluster/nodelocaldns.md
**removed**	content/zh/docs/tasks/administer-cluster/out-of-resource.md
**removed**	content/zh/docs/tasks/administer-cluster/highly-available-master.md
```

- [x] (9) admin 4  @Steven-my

```
2	12	content/zh/docs/tasks/administer-cluster/dns-custom-nameservers.md
3	3	content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md
3	5	content/zh/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md
14	4	content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md
7	0	content/zh/docs/tasks/administer-cluster/dns-debugging-resolution.md
```

- [x] (10) admin 5  @chenxuc  #29742

```
2	2	content/zh/docs/tasks/administer-cluster/access-cluster-services.md
3	0	content/zh/docs/tasks/administer-cluster/certificates.md
2	9	content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md
5	5	content/zh/docs/tasks/administer-cluster/topology-manager.md
7	3	content/zh/docs/tasks/administer-cluster/sysctl-cluster.md
15	13	content/zh/docs/tasks/administer-cluster/securing-a-cluster.md
```

- [x] (11) admin 6  @chenxuc   #29584

```
1	1	content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md
1	1	content/zh/docs/tasks/administer-cluster/change-pv-reclaim-policy.md
17	9	content/zh/docs/tasks/administer-cluster/controller-manager-leader-migration.md
14	1	content/zh/docs/tasks/administer-cluster/cpu-management-policies.md
1	1	content/zh/docs/tasks/administer-cluster/access-cluster-api.md
```

- [x] (12) access  @chenxuc  #29545 

```
2	9	content/zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md
4	4	content/zh/docs/tasks/access-application-cluster/list-all-running-container-images.md
26	20	content/zh/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md
```

- [x] (13) misc  @chenxuc  #29501

```
6	5	content/zh/docs/tasks/tls/certificate-rotation.md
3	3	content/zh/docs/tasks/configmap-secret/managing-secret-using-kustomize.md
3	3	content/zh/docs/tasks/configmap-secret/managing-secret-using-config-file.md
15	17	content/zh/docs/tasks/manage-daemon/update-daemon-set.md
1	1	content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md
8	4	content/zh/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md
1	1	content/zh/docs/tasks/extend-kubernetes/configure-aggregation-layer.md
```

- [x] (14) Indexed parallel processing  @howieyuen   #29816 

```
NEW   content/zh/docs/tasks/job/indexed-parallel-processing-static.md
```

### PR DESCRIPTION
**Pull request by steven-my**
related: #29329 
Translation for the run-app section in issue 29329.
Modified:
content/zh/docs/tasks/debug-application-cluster/audit.md
content/zh/docs/tasks/debug-application-cluster/debug-cluster.md
content/zh/docs/tasks/debug-application-cluster/debug-running-pod.md
content/zh/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md
content/zh/docs/tasks/run-application/delete-stateful-set.md
content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md

### CODE REVIEW

**Comment by tengqm** [comment]
@steven-my Please remove 'Fixes #29329' from your commit message and recommit this PR. This PR is related to #29329 but it is NOT supposed to close it. You can use `related: #29329` to link this PR to the umbrella issue.

File: content/zh/docs/tasks/debug-application-cluster/debug-running-pod.md [hunk 1]
```diff
@@ -234,7 +231,7 @@ You can view the state of the newly created ephemeral container using `kubectl d
 
 {{< note >}}
 {{< glossary_tooltip text="容器运行时" term_id="container-runtime" >}}必须支持`--target`参数。
-如果不支持，则临时容器可能不会启动，或者可能使用隔离的进程命名空间启动。
+如果不支持，则临时容器可能不会启动，或者可能使用隔离的进程命名空间启动以便 "ps" 不在其他容器内显示进程。
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/debug-running-pod.md:234]:
```suggestion
如果不支持，则临时容器可能不会启动，或者可能使用隔离的进程命名空间启动，
以便 "ps" 不显示其他容器内的进程。
```

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 2]
```diff
@@ -367,27 +367,27 @@ The detailed documentation of `kubectl autoscale` can be found [here](/docs/refe
...
+for setting the `replicas` of the underlying ReplicaSets so that they add up to a suitable
+number during the rollout and also afterwards.
 -->
 ## 滚动升级时扩缩   {#autoscaling-during-rolling-update}
 
-目前在 Kubernetes 中，可以针对 ReplicationController 或 Deployment 执行
-滚动更新，它们会为你管理底层副本数。
-Pod 水平扩缩只支持后一种：HPA 会被绑定到 Deployment 对象，
-HPA 设置副本数量时，Deployment 会设置底层副本数。
+Kubernetes 允许你在 Deployment 时执行滚动更新。在这种情况下，由 Deployment 为你管理下层的 ReplicaSet。
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md:380]:
```suggestion
Kubernetes 允许你在 Deployment 上执行滚动更新。在这种情况下，Deployment 为你管理下层的 ReplicaSet。
```

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 3]
```diff
@@ -367,27 +367,27 @@ The detailed documentation of `kubectl autoscale` can be found [here](/docs/refe
...
+当你为一个 Deployment 配置扩缩时，你为单个 Deployment 绑定一个 HorizontalPodAutoscaler。
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md:381]:
```suggestion
当你为一个 Deployment 配置自动扩缩时，你要为每个 Deployment 绑定一个 HorizontalPodAutoscaler。 
```

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 4]
```diff
@@ -367,27 +367,27 @@ The detailed documentation of `kubectl autoscale` can be found [here](/docs/refe
...
 -->
 ## 滚动升级时扩缩   {#autoscaling-during-rolling-update}
 
-目前在 Kubernetes 中，可以针对 ReplicationController 或 Deployment 执行
-滚动更新，它们会为你管理底层副本数。
-Pod 水平扩缩只支持后一种：HPA 会被绑定到 Deployment 对象，
-HPA 设置副本数量时，Deployment 会设置底层副本数。
+Kubernetes 允许你在 Deployment 时执行滚动更新。在这种情况下，由 Deployment 为你管理下层的 ReplicaSet。
+当你为一个 Deployment 配置扩缩时，你为单个 Deployment 绑定一个 HorizontalPodAutoscaler。 
+HorizontalPodAutoscaler 管理 Deployment 的 “replicas” 字段。
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md:382]: 在 markdown 中，反引号是用来以等宽字体显示变量或代码的，不要改成双引号！

```suggestion
HorizontalPodAutoscaler 管理 Deployment 的 `replicas` 字段。 
```

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 5]
```diff
@@ -367,27 +367,27 @@ The detailed documentation of `kubectl autoscale` can be found [here](/docs/refe
...
 ## 滚动升级时扩缩   {#autoscaling-during-rolling-update}
 
-目前在 Kubernetes 中，可以针对 ReplicationController 或 Deployment 执行
-滚动更新，它们会为你管理底层副本数。
-Pod 水平扩缩只支持后一种：HPA 会被绑定到 Deployment 对象，
-HPA 设置副本数量时，Deployment 会设置底层副本数。
+Kubernetes 允许你在 Deployment 时执行滚动更新。在这种情况下，由 Deployment 为你管理下层的 ReplicaSet。
+当你为一个 Deployment 配置扩缩时，你为单个 Deployment 绑定一个 HorizontalPodAutoscaler。 
+HorizontalPodAutoscaler 管理 Deployment 的 “replicas” 字段。 
+Deployment Controller 负责设置下层 ReplicaSet 的 “replicas”字段， 以便于在滚动及后续过程中为其增加到适当的数量。
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md:383]:
```suggestion
Deployment Controller 负责设置下层 ReplicaSet 的 `replicas`字段， 
以便确保在上线及后续过程副本个数合适。
```

File: content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 6]
```diff
@@ -367,27 +367,27 @@ The detailed documentation of `kubectl autoscale` can be found [here](/docs/refe
...
-The reason this doesn't work is that when rolling update creates a new replication controller,
-the Horizontal Pod Autoscaler will not be bound to the new replication controller.
+If you perform a rolling update of a StatefulSet that has an autoscaled number of
+replicas, the StatefulSet directly manages its set of Pods (there is no intermediate resource
+similar to ReplicaSet).
 -->
-通过直接操控副本控制器执行滚动升级时，HPA 不能工作，
-也就是说你不能将 HPA 绑定到某个 RC 再执行滚动升级。
-HPA 不能工作的原因是它无法绑定到滚动更新时所新创建的副本控制器。
+如果你对一个 replicas 值被自动扩缩过的 StatefulSet 执行滚动更新， 这个 StatefulSet 会直接管理他的 Pod 集合 （不存在类似 ReplicaSet 的中间资源）。
```
Comment (tengqm) [inline content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md:390]: 超长行注意折行。

```suggestion
如果你对一个副本个数被自动扩缩的 StatefulSet 执行滚动更新， 该 StatefulSet
会直接管理它的 Pod 集合 （不存在类似 ReplicaSet 这样的中间资源）。
```

**Comment by steven-my** [comment]
Hi @tengqm , thanks for review my changes. I modified files as your comments.

**Comment by reylejano** [comment]
/retitle [zh] translation for the run-app section

**Comment by tengqm** [comment]
Thanks.
/approve

**Pull request merged**